openmc/deplete/abc.py

```diff
@@ -0,0 +1,142 @@
+"""function module.
+
+This module contains the Operator class, which is then passed to an integrator
+to run a full depletion simulation.
+"""
+
+from collections import namedtuple
+import os
+from pathlib import Path
+from abc import ABCMeta, abstractmethod
+
+from .chain import Chain
+
+OperatorResult = namedtuple('OperatorResult', ['k', 'rates'])
+OperatorResult.__doc__ = """\
+Result of applying transport operator
+
+Parameters
+----------
+k : float
+    Resulting eigenvalue
+rates : openmc.deplete.ReactionRates
+    Resulting reaction rates
+
+"""
+try:
+    OperatorResult.k.__doc__ = None
```

Comment:
Why specifically are we setting ``__doc__`` to none?

Reply:
There is a default docstring on these properties for namedtuples that look something like "Alias for field number N". Since the main docstring already documents what the namedtuple stores, these property-specific docstrings are redundant and look weird in the generated documentation.